packages/botonic-plugin-hubtype-analytics/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@botonic/plugin-hubtype-analytics",
+  "version": "0.22.0-alpha.3",
+  "description": "Plugin for tracking in the Hubtype backend to see the results in the Hubtype Dashbord",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
+  "scripts": {
+    "build": "rm -rf lib && ../../node_modules/.bin/tsc -p tsconfig.json && ../../node_modules/.bin/tsc -p tsconfig.esm.json",
+    "build:watch": "npm run build -- --watch",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepublishOnly": "rm -rf lib && npm run build",
+    "lint": "npm run lint_core -- --fix",
+    "lint_ci": "npm run lint_core -- -c ../.eslintrc_slow.js",
+    "lint_core": "../../node_modules/.bin/eslint_d --cache --quiet 'src/**/*.ts*'"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.21.0",
+    "@botonic/core": "^0.22.0",
+    "@types/axios": "^0.14.0",
```

Comment:
is `@types/axios` in a different version than the module itself?

Reply:
I have removed the aryan types, they are not necessary